Add notes about admin playground (#1140)
saleor/saleor-docs: docs/api-usage/developer-tools.mdx

![Playground](./api-playground.png)

#### Admin authenticated playgrounds

> In the admin dashboard use `CMD+'` this would open playground that would have admin rights of your respective user.

Reviewer: What are the instructions for Windows/Linux users? 😄 

```suggestion
In the admin dashboard use `CMD+'`.  This will open playground that has admin rights of your respective user.
```

Reviewer 2: it becomes Ctrl when it os not mac user, team covered that case

#### Front-end playgrounds

> Each Saleor server comes with `/graphql/` endpoint. See [example](https://storefront1.saleor.io/graphql/).

Reviewer: The link doesn't work. Maybe we should remove this?
